avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -386,21 +449,50 @@ public void testFromConnectComplexWithDefaults() {
         .set("bytes", ByteBuffer.wrap("foo".getBytes()))
         .set("array", Arrays.asList("a", "b", "c"))
         .set("map", Collections.singletonMap("field", 1))
+        .set("date", dateDefVal)
+        .set("time", timeDefVal)
+        .set("ts", tsDefVal)
+        .set("decimal", decimalDefVal)
         .build();
 
-    org.apache.avro.generic.GenericRecord convertedAvroRecord = (org.apache.avro.generic
-        .GenericRecord) convertedRecord;
-    assertSchemaEquals(avroSchema, convertedAvroRecord.getSchema());
-    assertSchemaEquals(avroRecord.getSchema(), convertedAvroRecord.getSchema());
+    SchemaAndValue schemaAndValue = new SchemaAndValue(schema, struct);
+    schemaAndValue = convertToConnect(avroSchema, avroRecord, schemaAndValue);
+    schemaAndValue = convertToConnect(avroSchema, avroRecord, schemaAndValue);
+    schemaAndValue = convertToConnect(avroSchema, avroRecord, schemaAndValue);
+    assertNotNull(schemaAndValue);
+  }
```

Comment:
The previous few lines simply perform the conversion repeatedly using the outputs from the previous call. I could have made this a loop, but this makes it more clear which call fails if one does.